source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputTimeRenderer.mm

```diff
@@ -37,6 +37,10 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
     ACRDateTextField *field = [[ACRDateTextField alloc] initWithTimeDateInput:timeInput dateStyle:NSDateFormatterNoStyle];
 
     ACRInputLabelView *inputLabelView = [[ACRInputLabelView alloc] initInputLabelView:rootView acoConfig:acoConfig adptiveInputElement:timeInput inputView:field viewGroup:viewGroup dataSource:nil];
+
+    field.isAccessibilityElement = YES;
```

Comment:
Question: We seem to be repeating this across teh various input elements - isnt it possible to update the base input class with these changes so all derived elements as well as custom elements potentially, are able to get these changes applied automatically?

Reply:
With the merge with the branch for #4912, this is resolved.